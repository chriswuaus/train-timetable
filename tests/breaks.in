Paterson::Werris Creek::23:22:23 

Paterson::Werris Creek::00:01:23 
Paterson::Werris Creek::00:55:23 

Paterson::Lochinvar::01:06:11 
Paterson::Lochinvar::02:01:11 







Paterson::Lochinvar::02:56:11 
Condobolin::Wangaratta::11:56:26 

Condobolin::Wangaratta::12:01:26 
Condobolin::Wangaratta::12:58:26 

Condobolin::Maitland::09:30:22 

Condobolin::Maitland::10:01:22 