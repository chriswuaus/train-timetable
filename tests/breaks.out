The next train to Wangaratta from Condobolin departs at 12:58:26
